frontend/src/components/tables/BaseDisplayTable.vue

```diff
@@ -318,6 +322,30 @@ const getTooltipContent = (
                 class="font-bold text-lg lg:text-xl text-primary hover:text-primaryHover hover:underline"
                 ><Icon icon="tabler:trash"
               /></a>
+              <a
+                @click.prevent="approveButton?.callBack({ ...row })"
```

Comment:
this is just a comment but you can see here how my original design of this is not flexible ... hardcoding the buttons here is alright if we have a set number but as it expands more code